testing/import_text_test.py

```diff
@@ -148,7 +148,7 @@ def test_rowHash(self):
         self.assertTableResultEquals(
             mldb.query("select count(*) from titanic_hashed"),
             [["_rowName","count(*)"],
-             ["[]",279]]
+             ["[]",287]]
```

Comment:
ordering of rowHash (I replaced a hash function which is extremely slow on ARM with one which has a resonable speed; hence rowNames change)